app/assets/javascripts/govuk_publishing_components/lib/govspeak/govspeak-track-links.js

```diff
@@ -0,0 +1,30 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (GOVUK) {
```

Comment:
The reason why we needed GOVUK rather than `GOVUK.Modules` is that we need access to GOVUK.Analytics.

Happy to rename the other stuff to keep it consistent.

Reply:
Where does the code need access to `GOVUK.Analytics`? Why can't it access `window.GOVUK.Analytics`? Sorry, I'm not trying to be problematic, I'm genuinely interested.

Reply:
You're right @andysellick - sorry forgot about the fact that I could use window.GOVUK..

I've updated it now to remove that and make it more consistent like you suggested.